src/main/java/nl/hsac/fitnesse/util/RandomUtil.java

```diff
@@ -36,4 +36,14 @@ public String randomString(String permitted, int length) {
         }
         return result.toString();
     }
+
+    /**
+     * Return a random value from a given enum
+     * @param clazz The enum class to get a value from
+     * @return a randomly selected value from the enum
+     */
+    public <T extends Enum<?>> T randomEnum(Class<T> clazz){
+        int x = random.nextInt(clazz.getEnumConstants().length);
```

Comment:
randomElement is a method @keetron added recently that selects an element from an array. He uses it with an enum's `values()`, which is an array